content/pt-br/application_programming_interface.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Interface de programação de aplicações
```

Comment:
Interface de programação de aplicações to Interface de Programação de Aplicações